refactor(orc8r): replace deprecated errors module

wolfseb opened the pull request:
Replace functions from the github.com/pgk/errors package with
`fmt.Errorf`. Created in pairing with Moritz Huebner.

Signed-off-by: Sebastian Wolf <sebastian.wolf@tngtech.com>


## Summary

This PR implements the proposed solutions of ticket #12632 for the orc8r module.

A few instances of broken rollback error handling next to the changes
in this PR are also fixed.

## Test Plan

Perform Unit tests

## Additional Information

Worked in pairing with @MoritzThomasHuebner 

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

alexzurbonsen commented on orc8r/cloud/go/services/metricsd/servicers/southbound/servicer.go:
```diff
@@ -15,9 +15,9 @@ package southbound
 
 import (
 	"context"
+	"errors"
```
The cases in which std lib "errors" is imported without obvious use in the diff, are due to replacing errors.New from pkg/errors by errors.New from std lib, I assume?

wolfseb replied:
Yes, exactly. In the same file you should find a removal of github.com/pkg/errors

alexzurbonsen reviewed (approved):
Code changes look good to me.

maxhbr commented on orc8r/cloud/go/services/configurator/mconfig/marshal.go:
```diff
@@ -27,7 +28,7 @@ func MarshalConfigs(configs map[string]proto.Message) (ConfigsByKey, error) {
 	for k, v := range configs {
 		anyVal, err := ptypes.MarshalAny(v)
 		if err != nil {
-			return nil, errors.WithStack(err)
```
We have no replacement for `errors.WithStack`?

sebathomas replied:
No, you still get the file name and line but no call stack.

maxhbr reviewed (approved):
LGTM

Commit:
refactor(orc8r): replace deprecated errors module

Replace functions from the github.com/pgk/errors package
with `fmt.Errorf` in /magma/orc8r. Created in pairing
with Moritz Huebner.

Signed-off-by: Sebastian Wolf <sebastian.wolf@tngtech.com>

Commit:
refactor(orc8r): replace deprecated errors module

Replace functions from the github.com/pgk/errors package
with `fmt.Errorf` in /magma/lte. Created in pairing
with Moritz Huebner.

Signed-off-by: Sebastian Wolf <sebastian.wolf@tngtech.com>
